### PR TITLE
v4 fluent naming config, fix javadoc in non-string enum

### PR DESCRIPTION
**Pull request by weidongxu-microsoft**
@yungezz @ChenTanyi @xccc-msft 

Note this will force it for all places, regardless what casing used in spec.

### CODE REVIEW

**Comment by weidongxu-microsoft** [comment]
ManagedClusterAPIServerAccessProfile -> ManagedClusterApiServerAccessProfile

More in network since this is for all.

**Comment by weidongxu-microsoft** [comment]
Sample javadoc

```
    /**
     * De-serializes the instance to int value.
     *
     * @return the int value.
     */
    @JsonValue
    public int toInt() {
        return this.value;
    }
```

File: fluentnamer/readme.md [hunk 1]
```diff
@@ -19,7 +19,12 @@ pipeline:
     flatten-models: true
     flatten-payloads: true
     naming:
-      preserve-uppercase-max-length: 4
+      override:
+        api: Api
+        cpk: Cpk
+        sas: Sas
+        url: Url
+        xml: Xml
```
Comment (ChenTanyi) [inline fluentnamer/readme.md:27]: `SAS, XML, HTTP, JSON, API, URL, CPK`, it seems there is `Http` and `Json`

Comment (weidongxu-microsoft) [reply fluentnamer/readme.md:27]: I've deleted the `preserve-uppercase-max-length: 4` line. So all 4 letter will be changed.

**Pull request merged**